protoc-gen-gogo/generator/generator.go

```diff
@@ -1865,7 +1864,7 @@ var methodNames = [...]string{
 }
 
 // Generate the type and default constant definitions for this Descriptor.
-func (g *Generator) generateMessage(message *Descriptor) {
+func (g *Generator) generateMessage(file *FileDescriptor, message *Descriptor) {
```

Comment:
Are you sure we can't get the FileDescriptor given the Generator and the Descriptor.
Would be nice if we don't need to change this function signature.